### DISCUSSION TITLE
Any Future Plan

### DISCUSSION THREAD

**Discussion by xcfcode** [NumberChiffre/mcts-llm#4] [General]
Thanks for this neat repo! Any further plan to include more methods?

**Comment by NumberChiffre** [comment]
> Thanks for this neat repo! Any further plan to include more methods?

I'd be looking to update a few more papers here soon [**looking at SC-MCTS currently**]:
https://github.com/NumberChiffre/mcts-llm?tab=readme-ov-file#paper-implementations

Any specific papers/methodologies you are interested in? You are welcomed to add implementations :)
